src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java

```diff
@@ -313,6 +331,8 @@ public static String getRandomHydrateBreakMessageForPersonality(HydrateReminderP
                 break;
             case SANTA:
                 breakMessage = getRandomBreakMessage(HYDRATE_BREAK_SANTA_TEXT_LIST);
+            case KAWAII:
```

Comment:
Without a break statement for the `SANTA` case, the breakMessage will be overwritten with the `KAWAII` message instead.